<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectLayersDockWidgetForm</class>
 <widget class="QDockWidget" name="ConnectLayersDockWidgetForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>332</width>
    <height>713</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="focusPolicy">
   <enum>Qt::ClickFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Connect Layers</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout_6">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="11">
         <widget class="QToolButton" name="m_boxToolButton">
          <property name="toolTip">
           <string>Box - Creates a window over the layer...</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="0" column="6">
         <widget class="QToolButton" name="m_upToolButton">
          <property name="toolTip">
           <string>View Up Connected Layer...</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="0" column="10">
         <widget class="QToolButton" name="m_flickToolButton">
          <property name="toolTip">
           <string>Flick - Switches between layers...</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="8">
         <widget class="Line" name="line_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QToolButton" name="m_drawToolButton">
          <property name="toolTip">
           <string>Draw Connected Layers...</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="0" column="7">
         <widget class="QToolButton" name="m_downToolButton">
          <property name="toolTip">
           <string>View Down Connected Layer...</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="2">
         <widget class="QToolButton" name="m_updateToolButton">
          <property name="toolTip">
           <string>Update Tree Layers...</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="Line" name="line_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QToolButton" name="m_groupFoderToolButton">
          <property name="toolTip">
           <string>Group Folder Layers (needed draw action)... </string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QToolButton" name="m_activeToolButton">
          <property name="toolTip">
           <string>Enables layers connection...</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QTreeWidget" name="m_treeWidget">
          <column>
           <property name="text">
            <string>Layers</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>H</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSlider" name="m_horizontalSlider">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>V</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSlider" name="m_verticalSlider">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>A</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSlider" name="m_alphaSlider">
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="value">
           <number>255</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>m_activeToolButton</tabstop>
  <tabstop>m_drawToolButton</tabstop>
  <tabstop>m_updateToolButton</tabstop>
  <tabstop>m_groupFoderToolButton</tabstop>
  <tabstop>m_upToolButton</tabstop>
  <tabstop>m_downToolButton</tabstop>
  <tabstop>m_flickToolButton</tabstop>
  <tabstop>m_boxToolButton</tabstop>
  <tabstop>m_treeWidget</tabstop>
  <tabstop>m_horizontalSlider</tabstop>
  <tabstop>m_verticalSlider</tabstop>
  <tabstop>m_alphaSlider</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
